driver/config/provider.go

```diff
@@ -428,3 +429,7 @@ func (p *Provider) CGroupsV1AutoMaxProcsEnabled() bool {
 func (p *Provider) GrantAllClientCredentialsScopesPerDefault() bool {
 	return p.p.Bool(KeyGrantAllClientCredentialsScopesPerDefault)
 }
+
+func(p *Provider) RefreshTokenRotationGracePeriod() time.Duration {
+	return p.p.DurationF(KeyRefreshTokenRotationGracePeriod, 0)
```

Comment:
Should we limit to prevent this form being set to e.g. 1 year?

Reply:
Yes. I consider `<=1h`  extremely reasonable, what do you think ? In our deployment I would set this to 5-10m

Reply:
Done!